package.json

```diff
@@ -25,7 +25,9 @@
     "supertest-as-promised": "2.0.1",
     "underscore": "^1.7.0",
     "vend-nodejs-sdk": "^1.0.13",
-    "winston": "^0.8.3"
+    "winston": "^0.8.3",
+    "xlsx": "~0.8.0",
+    "excel-stream": "~1.1.1"
```

Comment:
use absolute version numbers, it help lock things down and keep them predictable